core/process.py

```diff
@@ -368,7 +368,7 @@ def _ApplyRedirect(self, r):
         except OSError as e:
           self.errfmt.Print_("Can't open %r: %s" %
                              (arg.filename, pyutil.strerror(e)),
-                             blame_loc=loc.Span(r.op_spid))
+                             blame_loc=r.op_loc)
```

Comment:
BTW I want to get rid of all the keyword arguments like blame_loc=

As mentioned they should be required arguments, which means they don't need keywords

The code is inconsistent now, but we can gradually change that